src/network/apiUrls.ts

```diff
@@ -15,5 +16,67 @@ export const newListUrl = (token: string, newList: string, boardId: string) => {
 };
 
 export const newCardUrl = (token: string, newCard: string, listId: string) => {
-  return `/1/cards?name=${newCard}&idList=${listId}&key=${API_KEY}&token=${token}`;
+  return `/1/cards?name=${newCard}&idMembers=${memberArr}&idList=${listId}&key=${API_KEY}&token=${token}`;
```

Comment:
why writing key and token again and again  create a custom func or constant for this